app/controllers/api/storage.php

```diff
@@ -950,9 +950,8 @@
         $cache = new Cache(new Filesystem(APP_STORAGE_CACHE . DIRECTORY_SEPARATOR . 'app-' . $project->getId() . DIRECTORY_SEPARATOR . $bucketId . DIRECTORY_SEPARATOR . $fileId)); // Limit file number or size
         $data = $cache->load($key, 60 * 60 * 24 * 30 * 3/* 3 months */);
 
+        $output = (empty($output)) ? ( empty($type) ? 'jpg' : $type ) : $output;
```

Comment:
Why are we assuming jpeg? What happens if the file is not an image? Maybe using the file mime-type to determine the default would be better.

Reply:
If the file is not an image, that part is already calculated above to get a default image preview.

Reply:
Using mime/type here to determine will again cause issue if the mime-type is not of an image.

Reply:
Agree, but if it is an image, we can tell if it's jpg, gif, or png.

Reply:
If it's an image, `$type` already has the type and it takes `$type` instead of default `jpg` ?